src/compile_cache.h

```diff
@@ -65,7 +65,9 @@ struct CompileCacheEnableResult {
 class CompileCacheHandler {
  public:
   explicit CompileCacheHandler(Environment* env);
-  CompileCacheEnableResult Enable(Environment* env, const std::string& dir);
+  CompileCacheEnableResult Enable(Environment* env,
+                                  const std::string& dir,
+                                  bool portable);
```

Comment:
Generally not a fan of bool arguments as they can be confusing at call sites. Can this be made an enum instead? e.g.

```cpp
enum class EnableOption {
  DEFAULT;
  PORTABLE;
};
CompileCacheEnableResult Enable(
    Environment* env,
    const std::string& dir,
    EnableOption option = EnableOption::DEFAULT);
```